bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/2D/Object.cpp

```diff
@@ -119,4 +59,22 @@ inline void OpenSpaceToolkitMathematicsPy_Geometry_2D_Object(pybind11::module &a
         .value("WKT", Object::Format::WKT)
 
         ;
+
+    // Create "object" python submodule
```

Comment:
Changes to this file are because we are combining 2D/Object.cpp and 2D/Objects.cpp